cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -195,7 +195,7 @@ func newKubeClient() (*kclient.Client, error) {
 		var err error
 		if config, err = kclientcmd.NewNonInteractiveDeferredLoadingClientConfig(
 			&kclientcmd.ClientConfigLoadingRules{ExplicitPath: *argKubecfgFile},
-			&kclientcmd.ConfigOverrides{ClusterInfo: kclientcmdapi.Cluster{Server: masterUrl, InsecureSkipTLSVerify: true}}).ClientConfig(); err != nil {
+			&kclientcmd.ConfigOverrides{ClusterInfo: kclientcmdapi.Cluster{Server: masterUrl}}).ClientConfig(); err != nil {
```

Comment:
@roberthbailey 

The current kubeconfigs made by kube-addons.sh have `clusters[].cluster.server` set to "http://kubernetes:443"`.  

When you say "control the apiserver location directly through the kubeconfig file", are you suggesting we would put a different value in that field, which might change over the lifetime of the cluster?  If so, clients will need to be restarted to pickup a new kubeconfig.  Seems less flexible than using a kubernetes service, which has a fixed service IP and can be re-targeted transparently.